x-pack/test/stack_functional_integration/configs/build_state.js

```diff
@@ -7,12 +7,16 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import dotEnv from 'dotenv';
 import testsList from './tests_list';
+import { fromNullable } from '../../../../src/dev/code_coverage/ingest_coverage/either';
 
 // envObj :: path -> {}
 const envObj = (path) => dotEnv.config({ path });
 
+const maybeUseExternalList = (obj) =>
+  fromNullable(obj.TESTS_LIST).fold(
```

Comment:
@LeeDr `fromNullable(obj.TESTS_LIST)` will test for an empty string and whether this value is defined at all.
If it does "not pass", the code defaults to using the testsLists fn in the kibana repo.